projector-server/src/main/kotlin/org/jetbrains/projector/server/idea/CaretInfoUpdater.kt

```diff
@@ -139,19 +126,24 @@ class CaretInfoUpdater(private val onCaretInfoChanged: (ServerCaretInfoChangedEv
           CaretInfo(point)
         }
 
+        val scrollBarWidth = if (visibleEditorRect.height < focusedEditorComponent.height) // check scrollbar should be visible
```

Comment:
Let's get rid of comments when we can ;)

```suggestion
        val visibleScrollBar = visibleEditorRect.height < focusedEditorComponent.height
        val scrollBarWidth = if (visibleScrollBar)
```

By the way, why do we compare heights but use the widths of the scrollbar? Is this check about the vertical or the horizontal scrollbar? It's not clear...

Reply:
We compare heights of editor to determine whether vertical scrollbar is visible and use its (scrollbar's) width. Added 'vertical' to variable names